lib/msal-common/src/index.ts

```diff
@@ -33,7 +35,8 @@ export { AuthorityType } from "./authority/AuthorityType";
 export { ProtocolMode } from "./authority/ProtocolMode";
 // Cache
 export { CacheManager, DefaultStorageClass } from "./cache/CacheManager";
-export { AccountCache, AccessTokenCache, IdTokenCache, RefreshTokenCache, AppMetadataCache, ValidCacheType, ValidCredentialType } from "./cache/utils/CacheTypes";
+export { AccountCache, AccessTokenCache, IdTokenCache, RefreshTokenCache, AppMetadataCache, ValidCacheType, 
+    ValidCredentialType } from "./cache/utils/CacheTypes";
```

Comment:
Why did some of these get pushed onto new lines?

Reply:
Just to improve readability so that you don't have to scroll a lot.

Reply:
I see. Can you set your IDE to wrap? Given everyone has different screen sizes this feels a bit arbitrary.

Reply:
Fixed